vipps-qr-api.md

```diff
@@ -313,3 +313,105 @@ does what one might expect, it deletes the QR. Once deleted, merchants can gener
 In addition to the `DELETE`-endpoint, it is also possible to add a `ttl`-attribute in the original `POST`-request. This attribute sets how many seconds the QR will live, before it is deleted permanently.
 
 Tip: If you want the same QR in different formats, perform `GET` calls on the same `id` with different `accept` headers and test what works best.
+
+## Merchant Callback QR codes
+
+Merchant callback QR's makes it possible for users to notify merchants that they want to pay with Vipps. It is the right solution for self-checkout, vending machines or similar situations where there is no casheer or buttons or other ways of letting the user communicate that they want to pay with Vipps.
+
+A Merchant callback QR is given a `merchantQrId` that the merchant is choosing upon creation. Then when the user scans the QR a callback is sent to the merchant containing this `merchantQrId`, so the merchant knows which QR code has been scanned and then can act accordingly. Most likely starting a payment towards the user.
+
+Here is an overview of the parameters that are specific to the Merchant Callback QR API.
```

Comment:
I think this should be "specific to the QR API", because I haven't heard of the "Merchant Callback QR API" or... is that something new?

Reply:
The Merchant callback QR Api is something new yea :)